fixed broken links, added community forum link, university link, spelling and  other mistakes (#13979)
scylladb/scylladb: docs/dev/docker-hub.md

![logo](http://www.scylladb.com/wp-content/uploads/mascot_medium.png)

> ScyllaDB Docker support x86_64 for all version, and aarch64 starting from ScyllaDB 4.6. Get it now with ScyllaDB [nightly build](https://hub.docker.com/r/scylladb/scylla-nightly)

Reviewer: ```suggestion
ScyllaDB Docker supports x86_64 for all versions, and aarch64 starting from ScyllaDB 4.6. Get it now with ScyllaDB [nightly build](https://hub.docker.com/r/scylladb/scylla-nightly)
```

## Quick start

> To startup a ScyllaDB single-node cluster in developer mode execute:

Reviewer: ```suggestion
To startup a ScyllaDB single-node cluster in developer mode, execute:
```

```console
$ docker run --name some-scylla --hostname some-scylla -d scylladb/scylla --smp 1
```

This command will start a ScyllaDB single-node cluster in developer mode
(see `--developer-mode 1`) limited by a single CPU core (see `--smp`).
Production grade configuration requires tuning a few kernel parameters
such that limiting the number of available cores (with `--smp 1`) is
the simplest way to go.

> Using multiple cores, requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.

Reviewer: ```suggestion
Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
```

You can use Docker volumes to improve the performance of ScyllaDB.

Create a Scylla data directory ``/var/lib/scylla`` on the host, which is used by the ScyllaDB container to store all the data:

```console
$ sudo mkdir -p /var/lib/scylla/data /var/lib/scylla/commitlog /var/lib/scylla/hints /var/lib/scylla/view_hints
```

Launch ScyllaDB using Docker's ``--volume`` command line option to mount the created host directory as a data volume in the container and disable ScyllaDB's developer mode to run I/O tuning before starting up the ScyllaDB node.

```console
$ docker run --name some-scylla --volume /var/lib/scylla:/var/lib/scylla -d scylladb/scylla --developer-mode=0
```

### Configuring resource limits

The ScyllaDB docker image defaults to running on overprovisioned mode and won't apply any CPU pinning optimizations, which it normally does in non-containerized environments.
For better performance, it is recommended to configure resource limits for your Docker container using the `--smp`, `--memory`, and `--cpuset` command line options, as well as 
disabling the overprovisioned flag as documented in the section "Command-line options".

### Restart ScyllaDB

> The Docker image uses supervisord to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using

Reviewer: ```suggestion
The Docker image uses a supervisor to manage ScyllaDB processes. You can restart ScyllaDB in a Docker container using:
```

Author: supervisord is the name of a system, I'll make it bold so it's more clear

### Related Links

* [Best practices for running ScyllaDB on docker](http://docs.scylladb.com/procedures/best_practices_scylla_on_docker/)

## User Feedback

### Issues

For bug reports, please use ScyllaDB's [issue tracker](https://github.com/scylladb/scylla/issues) on GitHub.
Please read the [How to report a ScyllaDB problem](http://docs.scylladb.com/operating-scylla/troubleshooting/report_scylla_problem/) page before you report bugs.

For general help, see ScyllaDB's [documentation](https://docs.scylladb.com/stable/).

> Training material and online courses can be found at [ScyllaDB University](https://university.scylladb.com/).

Reviewer: ```suggestion
You can find training material and online courses at [ScyllaDB University](https://university.scylladb.com/).
```

For questions and comments, use ScyllaDB's [Community Forum](https://forum.scylladb.com/).

### Contributing

Want to scratch your own itch and contribute a patch?
We are eager to review and merge your code.

> Please consult the [Contributing to ScyllaDB page](https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md)

Reviewer: ```suggestion
Please consult the [Contributing to ScyllaDB](https://github.com/scylladb/scylladb/blob/master/CONTRIBUTING.md) page.
```
